src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -306,4 +307,8 @@ private void appendShowHideButtons() {
   public void onFinish(ITestContext context) {
     CommonUtils.appendTextToFile(logPath, "</body></html>");
   }
+
+  public static void appendTextToLogFile(StringBuilder builder) {
+    CommonUtils.appendTextToFile(logPath, builder.toString());
```

Comment:
logPath is private, that is why i add method to add custom log to it, in my case i would add warning class to log line